packages/minimongo/minimongo.js

```diff
@@ -221,14 +221,26 @@ LocalCollection.Cursor.prototype.fetch = function () {
  * @locus Anywhere
  * @returns {Number}
  */
-LocalCollection.Cursor.prototype.count = function () {
+LocalCollection.Cursor.prototype.count = function (applySkipLimit) {
   var self = this;
+  // Have to respect `applySkipLimit`, so we remove limit & skip if `applySkipLimit`
```

Comment:
Could also modify `_getRawObjects(options)` to take `limit` and `skip` as options instead of using `self.limit`/`self.skip`, but that's a big change and not exactly necessary right now. Or just allow overriding of these values through `options`. Either way - not necessary, but could be a nice change.